intel/s2600/Makefile

```diff
@@ -225,6 +225,10 @@ geturoot:
 	go get -u github.com/u-root/u-root
 	go get -u github.com/u-root/cpu/...
 
+getmecleaner:
```

Comment:
This should not be here. It needs to be at the top level makefile, 
mainboards/Makefile
it applies to all boards.

Reply:
Take a look and you can see what i just changed.